en/flight_controller/cubepilot_cube_orange.md

```diff
@@ -136,11 +136,14 @@ Most users will not need to build this firmware!
 It is pre-built and automatically installed by *QGroundControl* when appropriate hardware is connected.
 :::
 
-To [build PX4](../dev_setup/building_px4.md) for this target:
+To [build PX4](../dev_setup/building_px4.md) for this target, open up the terminal and enter:
 ```
 make cubepilot_cubeorange
 ```
 
+:::warning
```

Comment:
Thanks very much. Note, I split out the note about VSCode because at some point that will disappear, but we will keep the information above.

Question, is there an issue number we can use to track a fix? If so, we'd include that. Reason being then it becomes easy to know when the note can be removed.

Arguably we might add a troubleshooting section in http://docs.px4.io/master/en/dev_setup/vscode.html too, but I think this is good for now. 

I'm going to merge, but can you please add a comment with the issue number here?

Reply:
Hmmm, I didn't see an issue opened for this guy here. I'll go ahead and open one.

Reply:
Thank you very much

Reply:
Ah, I see. No @Ionstream , I meant and issue against the platform in https://github.com/PX4/PX4-Autopilot/issues

Reply:
Oh my bad. Issue #17745 in https://github.com/PX4/PX4-Autopilot/issues

Reply:
Thanks. I added a note in https://github.com/PX4/PX4-user_guide/commit/9775c2c4c4c236919604a1c073115e99af1982f8 - we can remove that if/when the issue is fixed.